### PR TITLE
Docs/international text modification

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What type of PR is this?

- [ ] Feature
- [x] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## Which issue(s) this PR fixes:

Fixes # bugs caused by international text changed

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [x] No

### CODE REVIEW

**Comment by McDaddy** [comment]
if commits are doing the same thing with same title, better combine them together

**Comment by daskyrk** [comment]
labels ？

File: shell/app/locales/en.json [hunk 1]
```diff
@@ -3863,6 +3866,9 @@
     "visit domain": "visit domain",
     "waiting for deployment": "waiting for deployment"
   },
+  "shell": {
+    "common&#58;vip features tips": "common:vip features tips"
```
Comment (daskyrk) [inline shell/app/locales/en.json:3870]: should not have shell namespace, colon should not be escaped

**Pull request merged**